Dockerfile

```diff
@@ -13,12 +13,20 @@ WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
 
+ENV MYSQL_HOST mysql
+ENV MYSQL_PORT 3306
+ENV MYSQL_USER app_user
+ENV MYSQL_PASSWORD 1234
```

Comment:
Hardcoding sensitive information like `MYSQL_PASSWORD` is a security risk. Consider using environment variables or secret management tools to manage these credentials securely.